src/common/enums/podcast/podcastApiPath.ts

```diff
@@ -0,0 +1,12 @@
+enum PodcastsApiPath {
+  ROOT = '/',
+  $ID = '/:id',
+  SEARCH = '/search',
+  USERS = '/users',
+  USERS_$ID = '/users/:id',
+  INVITE = '/invite',
```

Comment:
There are useless paths here - INVITE for example

Reply:
thought we will use it in future, ok